xml/src/test/java/org/ehcache/xml/XmlConfigurationTest.java

```diff
@@ -176,6 +176,15 @@ public void testOneCacheConfig() throws Exception {
     assertThat(config.getCacheConfigurations().get("bar").getServiceConfigurations(), IsCollectionContaining.<ServiceConfiguration<?>>hasItem(instanceOf(FooConfiguration.class)));
   }
 
+  @Test
+  public void testAllExtension() {
+    Configuration config = new XmlConfiguration(XmlConfigurationTest.class.getResource("/configs/all-extensions.xml"));
+    assertThat(config.getServiceCreationConfigurations(), IsCollectionContaining.hasItem(instanceOf(BarConfiguration.class)));
```

Comment:
Can't you use a static import on hasItem? To help readability (or assertj ;-) )

Reply:
I used assertj in the new classes. This class already had junit asserts and that's why I kept it that way.